src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {
+    white,
+    black,
+    red,
+    green,
+    blue,
+    yellow,
+    brown

```

Comment:
```suggestion
    WHITE,
    BLACK,
    RED,
   ...

```

enum elements should be written using CAPS